src/extension.ts

```diff
@@ -145,6 +146,7 @@ export async function activate(context: vscode.ExtensionContext): Promise<APIBro
     const dependenciesProvider = new HelmDependencyDocumentProvider();
     const helmSymbolProvider = new HelmDocumentSymbolProvider();
     const completionProvider = new HelmTemplateCompletionProvider();
+    recommendExtensions(kubectl, context);
```

Comment:
Cosmetic: not sure this is the most natural place for this - this part of the code is mostly wiring up config.